joblog.go

```diff
@@ -12,15 +12,19 @@ import (
 
 type JobLog struct {
 	url         string
-	serviceName string
+	ServiceName string
```

Comment:
I think the serviceName is lowercase (made private), otherwise, there are two places where the serviceName can be modified, which is likely to cause confusion.

Reply:
ok

Reply:
🎉 🎉